test/plugin/scenarios/postgresql-scenario/support-version.list

```diff
@@ -0,0 +1,54 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+9.2-1002-jdbc4
+9.2-1003-jdbc3
+9.2-1003-jdbc4
+9.2-1004-jdbc3
+9.2-1004-jdbc4
+9.2-1004-jdbc41
+9.3-1100-jdbc3
+9.3-1100-jdbc4
+9.3-1100-jdbc41
+9.3-1101-jdbc3
+9.3-1101-jdbc4
+9.3-1101-jdbc41
+9.3-1102-jdbc3
+9.3-1102-jdbc4
+9.3-1102-jdbc41
+9.3-1103-jdbc3
+9.3-1103-jdbc4
+9.3-1103-jdbc41
+9.3-1104-jdbc4
+9.3-1104-jdbc41
+9.4-1200-jdbc4
+9.4-1200-jdbc41
+9.4-1201-jdbc4
+9.4-1201-jdbc41
+9.4-1202-jdbc4
+9.4-1202-jdbc41
+9.4-1202-jdbc42
+9.4-1203-jdbc4
+9.4-1203-jdbc41
+9.4-1203-jdbc42
+9.4-1204-jdbc4
+9.4-1204-jdbc41
+9.4-1204-jdbc42
+9.4-1205-jdbc4
+9.4-1205-jdbc41
+9.4-1205-jdbc42
+9.4-1206-jdbc4
+9.4-1206-jdbc41
+9.4-1206-jdbc42
```

Comment:
Also, make sure all these versions could pass, otherwise, you are facing PR CI failure, which is not satisfied with the requirements.

Reply:
ok, I will check those problem

Reply:
some jar doweload solw, any suggest?

Reply:
Download at Jenkins should be same, the VM is hosted in US Google Cloud.

Reply:
ok